### PR TITLE
Added Endpoint to retrieve Slides only image and order by an Admin

### PR DESCRIPTION
**Pull request by anavduo**
Implemented Mapstruct mapper

### CODE REVIEW

File: src/main/java/com/alkemy/ong/entity/SlideEntity.java [hunk 1]
```diff
@@ -2,19 +2,13 @@
...
 
 @Table(name = "slides")
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
-@SQLDelete(sql = "UPDATE slides SET softDeleted = true WHERE id=?") //This query provides the soft delete, as an update over slides
-@Where(clause = "softDeleted=false")
+@Data
```
Comment (JonathanBolatti) [inline src/main/java/com/alkemy/ong/entity/SlideEntity.java:11]: Ana, why did you remove the softDelete from SlideEntity? any special reason?

Comment (anavduo) [reply src/main/java/com/alkemy/ong/entity/SlideEntity.java:11]: Slide_entity
You are right! thanks for your comment, there was no attribute in the ticket but it should go, I will re added it

Comment (anavduo) [reply src/main/java/com/alkemy/ong/entity/SlideEntity.java:13]: Added the attribute and re added query and where clause, also updated main

**Comment by JonathanBolatti** [comment]
for my part it's approved

**Comment by anavduo** [comment]
SlidesDataBase
Slides_List
The complement documentation for the endpoint in Data Base and Postman test

**Comment by anavduo** [comment]
thanks Jonathan!
